docs/README.md

```diff
@@ -1,40 +1,105 @@
 
-![dag](source/_static/img/coproid_dag.png)
+iv. Start running your own analysis!
 
-## How to cite coproID
+```bash
+nextflow run maxibor/coproid --genome1 'GRCh37' --genome2 'CanFam3.1' --name1 'Homo_sapiens' --name2 'Canis_familiaris' --reads '*_R{1,2}.fastq.gz' --krakendb 'path/to/minikraken_db' -profile docker
+```
 
-The coproID article is coming.
+See [usage docs](docs/usage.md) for all of the available options when running the pipeline.
+
+## Documentation
+
+The nf-core/coproid pipeline comes with documentation about the pipeline, found in the `docs/` directory:
+
+The nf-core/coproid pipeline comes with documentation about the pipeline, found in the `docs/` directory and at the following address: [coproid.readthedocs.io](https://coproid.readthedocs.io)
+
+1. [Installation](https://nf-co.re/usage/installation)
+2. [Pipeline configuration
+    - [Local installation](https://nf-co.re/usage/local_installation)
+    - [Adding your own system config](https://nf-co.re/usage/adding_own_config)
+    - [Reference genomes](https://nf-co.re/usage/reference_genomes)
+3. [Running the pipeline](docs/usage.md)
+4. [Output and how to interpret the results](docs/output.md)
+5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
+
+## Credits
+
+nf-core/coproid was written by [Maxime Borry](https://github.com/maxibor).
+
+## Contributions and Support
+
+If you would like to contribute to this pipeline, please see the [contributing guidelines](.github/CONTRIBUTING.md).
+
+For further information or help, don't hesitate to get in touch on [Slack](https://nfcore.slack.com/channels/coproid) (you can join with [this invite](https://nf-co.re/join/slack)).
+
+## Citing
+
+coproID has been published in [peerJ](https://peerj.com/articles/9001). The bibtex citation is available below:
+
+```bibtex
+@article{borry_coproid_2020,
+ title = {{CoproID} predicts the source of coprolites and paleofeces using microbiome composition and host {DNA} content},
+ volume = {8},
+ issn = {2167-8359},
+ url = {https://peerj.com/articles/9001},
+ doi = {10.7717/peerj.9001},
+ language = {en},
+ urldate = {2020-04-20},
+ journal = {PeerJ},
+ author = {Borry, Maxime and Cordova, Bryan and Perri, Angela and Wibowo, Marsha and Honap, Tanvi Prasad and Ko, Jada and Yu, Jie and Britton, Kate and Girdland-Flink, Linus and Power, Robert C. and Stuijts, Ingelise and Salazar-García, Domingo C. and Hofman, Courtney and Hagan, Richard and Kagoné, Thérèse Samdapawindé and Meda, Nicolas and Carabin, Helene and Jacobson, David and Reinhard, Karl and Lewis, Cecil and Kostic, Aleksandar and Jeong, Choongwon and Herbig, Alexander and Hübner, Alexander and Warinner, Christina},
+ month = apr,
+ year = {2020},
+ note = {Publisher: PeerJ Inc.},
+ pages = {e9001}
+}
+```
+
+## Contributors
+
+[James A. Fellows-Yates](https://github.com/jfy133)
+
+## Tool references
```

Comment:
Based on your environment.yml, you're also missing

bedtools
vcftools
blast

I dnon't know if there is a convention the python packages, but if some projects need citations for funding, might be worth including them (might be overkill though)

Reply:
>bedtools
vcftools
blast

Removed them from the environment. No other citation needed as far as I can see.